src/index.ts

```diff
@@ -56,14 +58,14 @@ export default class Avalanche extends AvalancheCore {
   Health = () => this.apis.health as HealthAPI;
 
   /**
-     * Returns a reference to the Info RPC for a node.
+     * Returns a reference to the Index RPC for a node.
      */
-  Info = () => this.apis.info as InfoAPI;
+  Index = () => this.apis.index as IndexAPI;
 
   /**
-     * Returns a reference to the Index RPC for a node.
+     * Returns a reference to the Info RPC for a node.
      */
-  Index = () => this.apis.index as IndexAPI;
+  Info = () => this.apis.info as InfoAPI;
```

Comment:
Any particular reason for transposing these (and the imports above)? No need to change it, just curious.

Reply:
I wanted to make the alphabetically ordered